tests/Contacts.IntegrationTests/ContactsTest.cs

```diff
@@ -40,117 +41,187 @@ public async Task ShouldListNotOwnedChats()
         // arrange
         var bob = await _tester.SignInAsBob();
         await _tester.SignInAsAlice();
-        var (placeRootChatId, _) = await _tester.CreateChat(x => x with {
+
+        // non-place
+        var (publicChatId, publicChatInviteId) = await _tester.CreateChat(true);
+        var (privateChatId, privateChatInviteId) = await _tester.CreateChat(false);
+
+        // public place
+        var (publicPlaceRootChatId, _) = await _tester.CreateChat(x => x with {
```

Comment:
Better to add extension method CreatePlace(this IWebTester tester, bool isPublic)